airbyte-integrations/connectors/source-faker/source_faker/airbyte_message_with_cached_json.py

```diff
@@ -15,8 +15,3 @@ class AirbyteMessageWithCachedJSON(AirbyteMessage):
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        self._json = self.json(exclude_unset=True)
```

Comment:
this hack doesn't work anymore. we'll need to implement a custom serializer for the type in the CDK

Reply:
+1. - I agree that getting this connector on the modern CDK >= Speed.  For now anyway